.github/workflows/fast_testing.yml

```diff
@@ -10,6 +10,40 @@ on:
       - '*'
 
 jobs:
+  version_check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Setup tarantool
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: '2.9'
+
+      - name: Package version
+        # Found that Github checkout Actions pulls all the tags, but
+        # right it deannotates the testing tag, check:
+        #   https://github.com/actions/checkout/issues/290
+        # But we use 'git describe ..' calls w/o '--tags' flag and it
+        # prevents us from getting the needed tag for packages version
+        # setup. To avoid of it, let's fetch it manually, to be sure
+        # that all tags will exists always. The actions/checkout depth
+        # does not help.
+        run: |
+          git fetch --tags -f
+          tarantoolctl rocks make
+          echo HARDCODED_VERSION=$(tarantool -e "print(require('queue')._VERSION)")
```

Comment:
And I don't see any reason to describe version here as "hardcoded" too. The only thing where this work is relevant is a implementation description in the commit message.

Reply:
Deleted all `hardcoded` mentions